diff --git a/SConstruct b/SConstruct
index 920c754..0e736a6 100644
--- a/SConstruct
+++ b/SConstruct
@@ -2,7 +2,7 @@
 from misc.utility.scons_hints import *
 
 EnsureSConsVersion(4, 4)
-EnsurePythonVersion(3, 9)
+EnsurePythonVersion(3, 8)
 
 # System
 import glob
diff --git a/editor/gui/editor_about.cpp b/editor/gui/editor_about.cpp
index d65275e..323beeb 100644
--- a/editor/gui/editor_about.cpp
+++ b/editor/gui/editor_about.cpp
@@ -57,7 +57,7 @@ void EditorAbout::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_TRANSLATION_CHANGED: {
 			_about_text_label->set_text(
-					String(U"© 2014-present ") + TTR("Godot Engine contributors") + ".\n" +
+					String(U"© 2014-present ") + TTR("MindStudio Engine contributors") + ".\n" +
 					String(U"© 2007-2014 Juan Linietsky, Ariel Manzur.\n"));
 
 			_project_manager_label->set_text(TTR("Project Manager", "Job Title"));
@@ -211,7 +211,7 @@ Control *EditorAbout::_create_section(Control *p_parent, const String &p_title,
 }
 
 EditorAbout::EditorAbout() {
-	set_title(TTRC("Thanks from the Godot community!"));
+	set_title(TTRC("Thanks from the MindStudio community!"));
 	set_hide_on_ok(true);
 
 	VBoxContainer *vbc = memnew(VBoxContainer);
@@ -306,7 +306,7 @@ EditorAbout::EditorAbout() {
 	license_thirdparty->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	tc->add_child(license_thirdparty);
 
-	Label *tpl_label = memnew(Label(TTRC("Godot Engine relies on a number of third-party free and open source libraries, all compatible with the terms of its MIT license. The following is an exhaustive list of all such third-party components with their respective copyright statements and license terms.")));
+	Label *tpl_label = memnew(Label(TTRC("MindStudio Engine relies on a number of third-party free and open source libraries, all compatible with the terms of its MIT license. The following is an exhaustive list of all such third-party components with their respective copyright statements and license terms.")));
 	tpl_label->set_focus_mode(Control::FOCUS_ACCESSIBILITY);
 	tpl_label->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	tpl_label->set_autowrap_mode(TextServer::AUTOWRAP_WORD_SMART);
diff --git a/version.py b/version.py
index a203d93..40a7a40 100644
--- a/version.py
+++ b/version.py
@@ -1,9 +1,9 @@
-short_name = "godot"
-name = "Godot Engine"
+short_name = "Mindteco"
+name = "MindStudio"
 major = 4
 minor = 7
 patch = 2
 status = "stable"
 module_config = ""
-website = "https://godotengine.org"
+website = "http://www.mindteco.com"
 docs = "4.7"
