diff --git a/editor/editor_about.cpp b/editor/editor_about.cpp
index 6b592ba..3c7941a 100644
--- a/editor/editor_about.cpp
+++ b/editor/editor_about.cpp
@@ -171,7 +171,7 @@ ScrollContainer *EditorAbout::_populate_list(const String &p_name, const List<St
 }
 
 EditorAbout::EditorAbout() {
-	set_title(TTR("Thanks from the Godot community!"));
+	set_title(TTR("Thanks from the MindStudio community!"));
 	set_hide_on_ok(true);
 
 	VBoxContainer *vbc = memnew(VBoxContainer);
@@ -198,7 +198,7 @@ EditorAbout::EditorAbout() {
 	Label *about_text = memnew(Label);
 	about_text->set_v_size_flags(Control::SIZE_SHRINK_CENTER);
 	about_text->set_text(
-			String::utf8("\xc2\xa9 2014-present ") + TTR("Godot Engine contributors") + "." +
+			String::utf8("\xc2\xa9 2014-present ") + TTR("MindStudio Engine contributors") + "." +
 			String::utf8("\n\xc2\xa9 2007-2014 Juan Linietsky, Ariel Manzur.\n"));
 	version_info_vbc->add_child(about_text);
 
@@ -270,7 +270,7 @@ EditorAbout::EditorAbout() {
 	Label *tpl_label = memnew(Label);
 	tpl_label->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	tpl_label->set_autowrap_mode(TextServer::AUTOWRAP_WORD_SMART);
-	tpl_label->set_text(TTR("Godot Engine relies on a number of third-party free and open source libraries, all compatible with the terms of its MIT license. The following is an exhaustive list of all such third-party components with their respective copyright statements and license terms."));
+	tpl_label->set_text(TTR("MindStudio Engine relies on a number of third-party free and open source libraries, all compatible with the terms of its MIT license. The following is an exhaustive list of all such third-party components with their respective copyright statements and license terms."));
 	tpl_label->set_size(Size2(630, 1) * EDSCALE);
 	license_thirdparty->add_child(tpl_label);
 
diff --git a/version.py b/version.py
index 4b0e55c..9211244 100644
--- a/version.py
+++ b/version.py
@@ -1,9 +1,9 @@
-short_name = "godot"
-name = "Godot Engine"
+short_name = "Mindteco"
+name = "MindStudio"
 major = 4
 minor = 7
 patch = 2
 status = "stable"
 module_config = ""
-website = "https://godotengine.org"
+website = "http://www.mindteco.com"
 docs = "4.7"
